tests/FiberTraces.py

```diff
@@ -854,10 +794,12 @@ def suite():
 
     suites = []
     suites += unittest.makeSuite(FiberTraceTestCase)
-#    suites += unittest.makeSuite(tests.MemoryTestCase)
     return unittest.TestSuite(suites)
 
 def run(exit = False):
+    """Quiet down loggers which are too verbose"""
+    lsst.log.setLevel("CameraMapper", lsst.log.FATAL)
+    lsst.log.setLevel("afw.image.ExposureInfo", lsst.log.FATAL)
```

Comment:
Upgrade our LSST code first, then re-evaluate the need for changing log levels. In the mean time, either live with the noise or suppress it, depending on how long you expect we're going to live in the current state.